src/shared/codelens/csharpCodeLensProvider.ts

```diff
@@ -66,8 +66,8 @@ export async function initialize({
     localInvokeCommand = new DefaultSamLocalInvokeCommand(getChannelLogger(toolkitOutputChannel), [
         WAIT_FOR_DEBUGGER_MESSAGES.DOTNET
     ])
-}: CodeLensProviderParams): Promise<void> {
-    vscode.commands.registerCommand(getInvokeCmdKey(CSHARP_LANGUAGE), async (params: LambdaLocalInvokeParams) => {
+}: CodeLensProviderParams): Promise<vscode.Disposable> {
```

Comment:
I'm not a fan of the initialize method returning a disposable. Conceivably in the future, these methods could produce more than one disposable. Can the methods internally add the disposables to ext.extensionContext.subscriptions?